lib/cell.rb

```diff
@@ -1,9 +1,10 @@
 class Cell
-  attr_reader :coordinate, :ship
+  attr_reader :coordinate, :ship, :fired_upon
```

Comment:
This is partly my bad but could we add each attr_reaader to its own line?

Reply:
Good catch. Revised attr_reader to multiline syntax.